src/Client.cpp

```diff
@@ -218,6 +218,15 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    CefRefPtr<CefFrame> frame = browser->GetMainFrame();
+
+    frame->ExecuteJavaScript(
+        "window.triggerCefPdf = function () { window."
```

Comment:
You have this `request` parameter where you can pass an action, and inside cefpdf you use `jobManager->Abort(browser)` when `request == "abort"`